nemo/collections/nlp/modules/common/transformer/transformer_decoders.py

```diff
@@ -148,13 +148,23 @@ def __init__(
 
     def _get_memory_states(self, decoder_states, decoder_mems_list=None, i=0):
```

Comment:
Curious why decoder mems is still a list here?

Reply:
I can change the var name to just decoder_mems if you feel that's better. In general, I didn't change any variable names to ensure none of the existing code breaks. But I don't think decoder_mems_list for this particular function (get_memory_states) is being used as a named parameter anywhere - so can change it if you wish.